### PR TITLE
Update C# sample code

### PR DESCRIPTION
**Pull request by tiembo**
- Update ConsumerSurveys library name
- Update package versions
- Add C# project specific README
- Add C# directory specific .gitignore

### CODE REVIEW

**Comment by tiembo** [comment]
cc @mwerbos

File: client_libraries-DEPRECATED/csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Instructions for running the sample:
+
+1. Create a service account via the Google Developer Console
+  - Go to the [Google Developer Console](https://console.developers.google.com/apis/credentials)
+  - Click on "Create credentials", then "Service account key"
+  - Select .p12 as key format, and click "Create"
```
Comment (mwerbos) [inline client_libraries-DEPRECATED/csharp/README.md:6]: Do we have to use P12?

**Review by mwerbos — APPROVED** [review]
Please address whether using P12 is mandatory (since it's deprecated I'd rather avoid it); otherwise LGTM.

File: client_libraries-DEPRECATED/csharp/README.md (hunk 1 above)
Comment (tiembo) [reply client_libraries-DEPRECATED/csharp/README.md:6]: We don't; this pull request was mainly for a baseline set of instructions and to update the libraries.

I added an issue to update the code for JSON credentials and to deprecate p12 credentials: https://github.com/google/surveys/issues/15

**Pull request merged**